src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+
+  transform: scale(1);
+  transition: transform 300ms ease-in-out;
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color-1;
+    margin: 0;
+    padding-bottom: 4px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover .card__name {
+    color: #34568b;
+    transition: transform 300ms ease-in-out;
+  }
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    margin-left: 19px;
+  }
+
+  &__top {
+    margin: 0 16px;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $text-color-2;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 16px 16px;
+    display: flex;
+
+    &-inscription {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $text-color-2;
+    }
+
+    &-count {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $text-color-1;
+    }
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 16px;
+    margin-bottom: 24px;
+
+    &-response {
+      margin: 0;
+      font-size: 10px;
+      color: $text-color-1;
+      font-weight: 400;
+    }
+  }
+  &-button {
+    text-align: center;
+
+    &__link {
+      display: inline-block;
+      padding: 12px 69px;
+      background-color: $button-color;
+      color: $text-color-3;
+      font-weight: 700;
+      font-size: 14px;
+      line-height: 16px;
+      text-decoration: none;
+      border-radius: 5px;
+      text-transform: uppercase;
+
+      &:hover {
+        background-color: $text-color-3;
+        color: $button-color;
+        border: 1px solid $button-color;
+        padding: 12px 69px 11px;
```

Comment:
Avoid changing padding on hover to prevent shifting of the content. Instead, use 'box-shadow' to create the border effect on hover without affecting the box model. This will ensure the button doesn't change its size when hovered over.